internal/action/edit.go

```diff
@@ -30,7 +30,7 @@ func (s *Action) Edit(c *cli.Context) error {
 func (s *Action) edit(ctx context.Context, c *cli.Context, name string) error {
 	ed := editor.Path(c)
 	if err := editor.Check(ctx, ed); err != nil {
-		out.Red(ctx, "Failed to check editor config: %s", err)
+		out.Red(ctx, "❌ Failed to check editor config: %s", err)
```

Comment:
This would be best done using `out.Error`, no? Since it's an error and it would then print on stderr instead of stdout.

Reply:
actually it should be a warning, but otherwise: yes.